bal/blockchain.py

```diff
@@ -169,9 +224,14 @@ def do_register_peers():
     simulation_path = args.simulationpath
     if simulation_path != '':
         if args.variant.find('pos') == 0:
-            blockchain = POSBlockchainSimulation(p2p_port, initial_difficulty, simulation_path, args.name)
+            blockchain = POSBlockchainSimulation(
+                p2p_port, initial_difficulty, simulation_path, args.name)
         else:
-            blockchain = POWBlockchainSimulation(p2p_port, initial_difficulty, simulation_path, args.name)
+            blockchain = POWBlockchainSimulation(
+                p2p_port, 
+                initial_difficulty, 
+                simulation_path, 
+                args.name)
```

Comment:
not sure what PEP8 recommends but I feel like we need one more tab between `=` and parameters.(parameters should be lined up one tab after "=" sign?)